src/test/java/com/meilisearch/integration/KeysTest.java

```diff
@@ -28,7 +29,8 @@ static void cleanMeiliSearch() {
     /** Test Get Keys */
     @Test
     public void testClientGetKeys() throws Exception {
-        Key[] keys = client.getKeys();
+        Result<Key> result = client.getKeys();
```

Comment:
It shouldn't be `Result<Key[]>` ?

Reply:
No, because it is already a `<T> []` so it just needs the type of with what we will do the `[]`

Reply:
I see, I think it is easier to understand `Result<Key[]>` or `Results<Key>` or even `KeysResults` (because of the plural). 

This way I tend to believe I will deal with a only one object.

Reply:
Yes I had to change all `Result` by `Results`